requirements.txt

```diff
@@ -10,12 +10,12 @@
 # scripts/convert-requirements-to-conda-yml.py as though it can only be found
 # on pip.
 
-GDAL>=3.4.2,<3.7.0
+GDAL>=3.4.2,<3.6.0
```

Comment:
python 3.11 is incompatible with gdal<3.6, and our test suite is failing with a cryptic error with gdal>=3.6